diff --git a/i3/config b/i3/config
index 71e8a8c..cbca041 100644
--- a/i3/config
+++ b/i3/config
@@ -2,17 +2,17 @@
 
 exec_always --no-startup-id /home/tim/.config/i3/display-setup.sh
 
-workspace 1  output _METACONF_REPLACE_PRI_DISP_1 _METACONF_REPLACE_PRI_DISP_2
-workspace 2  output _METACONF_REPLACE_PRI_DISP_1 _METACONF_REPLACE_PRI_DISP_2
-workspace 3  output _METACONF_REPLACE_PRI_DISP_1 _METACONF_REPLACE_PRI_DISP_2
-workspace 4  output _METACONF_REPLACE_PRI_DISP_1 _METACONF_REPLACE_PRI_DISP_2
-workspace 5  output _METACONF_REPLACE_PRI_DISP_1 _METACONF_REPLACE_PRI_DISP_2
-
-workspace 6  output _METACONF_REPLACE_SEC_DISP_1 _METACONF_REPLACE_SEC_DISP_2
-workspace 7  output _METACONF_REPLACE_SEC_DISP_1 _METACONF_REPLACE_SEC_DISP_2
-workspace 8  output _METACONF_REPLACE_SEC_DISP_1 _METACONF_REPLACE_SEC_DISP_2
-workspace 9  output _METACONF_REPLACE_SEC_DISP_1 _METACONF_REPLACE_SEC_DISP_2
-workspace 10 output _METACONF_REPLACE_SEC_DISP_1 _METACONF_REPLACE_SEC_DISP_2
+workspace 1  output _METACONF_REPLACE_PRI_DISP
+workspace 2  output _METACONF_REPLACE_PRI_DISP
+workspace 3  output _METACONF_REPLACE_PRI_DISP
+workspace 4  output _METACONF_REPLACE_PRI_DISP
+workspace 5  output _METACONF_REPLACE_PRI_DISP
+
+workspace 6  output _METACONF_REPLACE_SEC_DISP
+workspace 7  output _METACONF_REPLACE_SEC_DISP
+workspace 8  output _METACONF_REPLACE_SEC_DISP
+workspace 9  output _METACONF_REPLACE_SEC_DISP
+workspace 10 output _METACONF_REPLACE_SEC_DISP
 
 # lock screen
 exec --no-startup-id $HOME/.config/i3/lock.sh -u $HOME/Pictures/Wallpaper/current.jpg
diff --git a/install.sh b/install.sh
index c2ecda8..2cee88e 100755
--- a/install.sh
+++ b/install.sh
@@ -22,7 +22,7 @@ for line in $(cat "metaconfig/$(hostname).metaconf"); do
 
 	KEY=$(echo $line | sed -n 's/^\([^ ]\+\) *= /\1/p')
 	VAL=$(echo $line | sed -n 's/^[^ ]\+ *= *\([^ ].*\)/\1/;
-	                           s/\([^\\]\)#.*/\1/;
+	                           s/\([^\\]\)#.*/\1/; s/^#.*//;
                                s/\\#/#/g; s/\s*$//gp')
 
 	if [ "$KEY" = '' ]; then
diff --git a/metaconfig/tim-laptop.metaconf b/metaconfig/tim-laptop.metaconf
index 2f5a307..dbea225 100644
--- a/metaconfig/tim-laptop.metaconf
+++ b/metaconfig/tim-laptop.metaconf
@@ -20,8 +20,8 @@ SEC_DISP = DP5  HDMI-0
 LOAD_CACHE_FILESYSTEMS =      # onedrive, dropbox
 LOAD_NETWORK_FILESYSTEMS = \# # google drive
 
-LOAD_TOUCHSCREEN_TOOLS = 
-
+LOAD_TOUCHSCREEN_TOOLS =      # My regex isn't very good, for these
+                              # to work they can't end on whitespace
 VOLUME_TICK_UP   = +5%
 VOLUME_TICK_DOWN = -5%
 
